sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -1064,6 +1064,18 @@ def __init__(self, defensive_copy=True):
     self.global_state = {}
     self.defensive_copy = defensive_copy
 
+  def copy(self):
+    cloned_object = copy.copy(self)
```

Comment:
<!--new_thread; commit:3f1a714b613353fdfe62e46e63c7bbfa47d98295; resolved:0-->
Let's omit the copy and just create a new object:

```python
cloned_object = InMemoryUnmergedState(defensive_copy=self.defensive_copy)
```